jetty-core/jetty-xml/src/main/java/org/eclipse/jetty/xml/XmlAppendable.java

```diff
@@ -30,31 +33,31 @@
     private final Stack<String> _tags = new Stack<>();
     private String _space = "";
 
-    public XmlAppendable(OutputStream out, String encoding) throws IOException
+    public XmlAppendable(OutputStream out, Charset charset) throws IOException
```

Comment:
This constructor is never used with a Charset that isn't UTF-8, should probably just remove this constructor too and have only 1 constructor that has only 1 parameter `OutputStream`.